ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriberTest.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.rest.server.resources.streaming;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.easymock.Capture;
+import org.easymock.CaptureType;
+import org.easymock.EasyMock;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Map;
+
+import javax.websocket.CloseReason;
+import javax.websocket.CloseReason.CloseCodes;
+import javax.websocket.RemoteEndpoint.Async;
+import javax.websocket.RemoteEndpoint.Basic;
+import javax.websocket.Session;
+
+import io.confluent.ksql.rest.entity.SchemaMapper;
+import io.confluent.ksql.rest.server.resources.streaming.Flow.Subscription;
+
+import static org.junit.Assert.assertEquals;
+
+public class WebSocketSubscriberTest {
+
+  @Test
+  public void testSanity() throws Exception {
+    Session session = EasyMock.mock(Session.class);
+    Async async = EasyMock.mock(Async.class);
+
+    WebSocketSubscriber<Map<String, Object>> subscriber = new WebSocketSubscriber<>(session, new ObjectMapper());
+
+    Subscription subscription = EasyMock.mock(Subscription.class);
+
+    subscription.request(1);
```

Comment:
Looks like from here to the reset is the same code in other tests. Extract a method to make the test clearer.

Reply:
done